internal/runners/packages/search.go

```diff
@@ -86,6 +80,56 @@ func targetedLanguage(languageOpt string) (string, *failures.Failure) {
 	return model.LanguageForCommit(proj.CommitUUID())
 }
 
+type modules []string
+
+func makeModules(normalizedName string, pack *model.IngredientAndVersion) modules {
+	var ms modules
+	for _, x := range pack.LatestVersion.ProvidedFeatures {
+		if x.Feature != nil && *x.Feature != normalizedName {
+			ms = append(ms, *x.Feature)
+		}
```

Comment:
```suggestion
	for _, pf := range pack.LatestVersion.ProvidedFeatures {
		if pf.Feature != nil && *pf.Feature != normalizedName {
			ms = append(ms, *pf.Feature)
		}
```

I'm sorry this is nitpicky but this is a new level of variable naming that just triggers my OCD. Short vars are one thing. Short vars that have zero relation to their value in their naming is a whole other thing.

Reply:
Not nitpicky! I will fix this - it's a scratch var name because I didn't know the name of the type/field that was needed when I first wrote the code. :facepalm: